action.yml

```diff
@@ -13,12 +13,14 @@ inputs:
       Path to write test results report in JUnit XML format.
     required: false
     default: ""
+
```

Comment:
Personally, I don't mind the extra newlines too too much. However, from what I've seen with other `action.yml` files from other actions (ex. [upload-artifact](https://github.com/actions/upload-artifact/blob/main/action.yml), [checkout](https://github.com/actions/checkout/blob/main/action.yml)), there usually aren't newlines between the input declarations 🤔 

Since we've rolled without those newlines in the other actions too, I think it should be fine to keep the consistency of not having spaces in this one too. Should we decide to update our formatting, we can do so broadly across all the repos in a separate PR 😄